homeassistant/components/modbus/quality_scale.yaml

```diff
@@ -0,0 +1,94 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling: done
+  brands:
+    status: exempt
+    comment: |
+      Modbus have no brand information.
+  common-modules: done
+  config-flow-test-coverage:
+    status: exempt
+    comment: |
+      Modbus is a protocol implementation and do not support config_flow.
+  config-flow:
+    status: exempt
+    comment: |
+      Modbus is a protocol implementation and do not support config_flow.
```

Comment:
I don't agree with that it does not support a config flow. MQTT is also a protocol integration and does support config flow now that we have config subentries

Reply:
Well Franck told me the other day, that it made sense not to use config_flow.

Try to reproduce my test setup, 5 connections each with between 200-300 entities....then you will see 2 things:
1) you loose the overview, compared to yaml
2) you get tired of copying the same parameters several 100 times....in yaml you simple do a copy/paste of a whole block, and just change slave: manually.

Reply:
Okay but that doesn't mean it doesn't support it.

On your points:
1. I don't see why that would be the case, and if that is a problem we can always look to improve something
2. MQTT is also doing efforts to making entities easy to create without much hassle

Reply:
> Sure, I am also fine with it being exempted, but then we should say that it is exempted because the rule allows it to, not to say its not supported, because that is untrue :)

That I agree on :)

```suggestion
  config-flow:
    status: exempt
    comment: |
      Modbus is a protocol implementation and thus is not required to implement a configuration flow as per ADR-0010.
```